### DISCUSSION TITLE
Starting with FIFO?

### DISCUSSION THREAD

**Discussion by Charles-Kaminski** [eprbell/rp2#103] [General]
I love the concept.  I would enjoy diving into the code and seeing if I can help.  But I do have a question first.  I'm surprised you started with FIFO and not HIFO (or at least LIFO) like so many services do.  Any reason not to implement HIFO or LIFO?  Any reason you started with FIFO?

**Comment by eprbell** [comment]
Thanks for the comment. LIFO and HIFO [have been implemented](https://github.com/eprbell/rp2/tree/main/src/rp2/plugin/accounting_method), however [an issue was brought up](https://github.com/eprbell/rp2/issues/79) that pertains to interpretation of IRS regulation and that caused us to [conservatively disable LIFO and HIFO](https://github.com/eprbell/rp2/blob/main/src/rp2/plugin/country/us.py#L37) until we could fully resolve the issue. Feel free to contribute to the discussion.

**Comment by Charles-Kaminski** [comment]
Thanks @eprbell.  I'll take a look at the discussion from issue 79

**Reply by eprbell** [reply]
The issue has been resolved and LIFO and HIFO have been reactivated. See #79 for more details.
